

message TestProtoMessage
{
    required string str = 1;
}